### PR TITLE
Gary

### PR DESCRIPTION
**Pull request by garywuuu**
Only include changes on index.html file.

### CODE REVIEW

**Comment by WilliamHYZhang** [comment]
Can you unstage the cache file and resolve the solidity conflict @garywuuu? Thanks!

**Comment by garywuuu** [comment]
Should be good now!

File: contract/cache/solidity-files-cache.json [hunk 1]
```diff
@@ -1,9 +1,15 @@
 {
   "_format": "hh-sol-cache-2",
   "files": {
+<<<<<<< HEAD
```
Comment (WilliamHYZhang) [inline contract/cache/solidity-files-cache.json]: Seems like there was an unresolved conflict here, best to just remove this cache file.

File: contract/contracts/CarbonCredit.sol [hunk 2]
```diff
@@ -444,6 +451,14 @@ contract VCCToken is ERC20 {
 			emit Verified(msg.sender, CO2);
 		}
 
+<<<<<<< HEAD
```
Comment (WilliamHYZhang) [inline contract/contracts/CarbonCredit.sol]: Please resolve conflicts

**Comment by garywuuu** [comment]
deleted cache file and i think contract should be resolved

**Comment by WilliamHYZhang** [comment]
Great job, thanks!

**Pull request merged**